scripts/golangci.yml.in

```diff
@@ -156,7 +156,71 @@ linters-settings:
     max-blank-identifiers: 3
   nestif:
     min-complexity: 7
-
+  varnamelen:
+    ignore-names:
```

Comment:
this list gets quite long, I am not sure that is properly addressing the linter concerns... Is there a plan to reduce this list?